Dockerfile

```diff
@@ -101,6 +117,11 @@ RUN pip install --no-cache-dir --upgrade pip setuptools wheel \
     pandas \
     pypdf2
 
+###
+# Remove install dependencies
+###
+# RUN apt-get remove --quiet --quiet $INSTALL_DEPS
+
```

Comment:
Is there a reason this remains?

```suggestion
```

Reply:
It remains because I am planning to sort out which dependencies are install deps and which are really needed, so the former can be removed after they are not needed.  This will reduce the size of the resulting Docker image.

Reply:
I'd like to suggest you do a multi-stage build if feasible at that point.